x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/data_view/static.spec.ts

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import expect from '@kbn/expect';
+import { DataView } from '@kbn/data-views-plugin/common';
+import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
+import request from 'superagent';
+import { getStaticDataViewId } from '@kbn/apm-data-view';
+import { SupertestWithRoleScope } from '../../../../services/role_scoped_supertest';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+import {
+  SupertestReturnType,
+  ApmApiError,
+} from '../../../../../../apm_api_integration/common/apm_api_supertest';
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
+  const roleScopedSupertest = getService('roleScopedSupertest');
+  const synthtraceService = getService('synthtrace');
+  const logger = getService('log');
+  const dataViewPattern =
+    'traces-apm*,apm-*,traces-*.otel-*,logs-apm*,apm-*,logs-*.otel-*,metrics-apm*,apm-*,metrics-*.otel-*';
+
+  function createDataViewWithWriteUser({ spaceId }: { spaceId: string }) {
+    return apmApiClient.writeUser({
+      endpoint: 'POST /internal/apm/data_view/static',
+      spaceId,
+    });
+  }
+
+  function createDataViewWithReadUser({ spaceId }: { spaceId: string }) {
+    return apmApiClient.readUser({
+      endpoint: 'POST /internal/apm/data_view/static',
+      spaceId,
+    });
+  }
+
+  describe('Data view static', () => {
+    let supertest: SupertestWithRoleScope;
+    before(async () => {
```

Comment:
we need an `after` hook to clean up the credentials created by `roleScopedSupertest`

```ts
after(async () => {
      await supertest.destroy();
    });
```